SUMMARY.md

```diff
@@ -7,6 +7,7 @@
 - [9/21: More HTML Elements](MoreHtml/StudentDesc.md)
   - [Restaurant Code-Along](MoreHtml/RestaurantCodeAlong.md)
 - [9/28: More HTML Part 2](MoreHtml2/StudentDesc.md)
+  - [Halloween Web Design](MoreHtml2/WebDesignRequirements/halloween.md)
```

Comment:
adding this link in this file is what actually allows the page to be viewed on the website